app/src/main/java/com/kevo/displaydemo/MainActivity.kt

```diff
@@ -103,11 +103,11 @@ class MainActivity : AppCompatActivity(), PresentationHelper.Listener {
     }
 
     override fun showPreso(display: Display) {
-        preso = BasePresentation(this, display)
-        preso?.show()
+        preso = SimplePresentationFragment(this, display)
+        preso!!.show(supportFragmentManager, SimplePresentationFragment.TAG)
     }
 
-    override fun clearPreso(showInLine: Boolean) {
+    override fun clearPreso() {
```

Comment:
❓ By the looks of it, the "dialog" is getting dismissed -- is this effectively the same thing as "clearing the secondary screen"?

Reply:
Not super familiar with the inner workings for how android handles the dismissing of a Presentation but AFAIK this will dismiss the Presentation and the secondary screen should return to it's default state of mirroring the primary screen.